<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portfolioStockWidget</class>
 <widget class="QWidget" name="portfolioStockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="tickerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>APPL</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPriceLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>8</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>3000.66</string>
    </property>
   </widget>
   <widget class="QLabel" name="changeLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>8</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>0.10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="boughtAtLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>8</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>3000.66</string>
    </property>
   </widget>
   <widget class="QLabel" name="sharesLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>8</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalValueLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>8</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>$3000.66</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
